policy/pipeline/task_bundle_test.rego

```diff
@@ -74,10 +74,7 @@ test_acceptable_bundle_up_to_date_maintained_version if {
 
 # Warn about out of date bundles that are still acceptable.
 test_acceptable_bundle_out_of_date_past if {
-	tasks := [
-		{"name": "my-task-1", "taskRef": {"bundle": "reg.com/repo@sha256:bcd"}},
-		{"name": "my-task-2", "taskRef": {"bundle": "reg.com/repo@sha256:cde"}},
```

Comment:
Given the list of `task_bundles`, `cde` is older than `bcd`. Usage of `cde` should cause a violation. Removed it here as this was clearly the wrong behavior.